schemas/institution.json

```diff
@@ -0,0 +1,56 @@
+{
+	"title": "Institution",
+	"description": "An institution, such as a university, that may be used to track usage statistics, handle subscriptions, and more.",
+	"required": [
+		"contextId",
+		"name"
+	],
+	"properties": {
+		"_href": {
+			"type": "string",
+			"description": "The URL to this institution in the REST API.",
+			"format": "uri",
+			"readOnly": true,
+			"apiSummary": true
+		},
+		"contextId": {
+			"type": "integer",
+			"apiSummary": true
+		},
+		"id": {
+			"type": "integer",
+			"readOnly": true,
+			"apiSummary": true
+		},
+		"name": {
+			"type": "string",
+			"multilingual": true,
+			"apiSummary": true,
+			"validation": [
+				"nullable"
+			]
+		},
+		"ror": {
+			"type": "string",
+			"description": "The ROR of this institution. See: https://ror.org/",
+			"apiSummary": true,
+			"validation": [
+				"nullable",
+				"regex:/^0[^ILOU]{6}\\d{2}$/"
```

Comment:
Users [should interact with RORs as URLs](https://ror.org/facts/):
> ![image](https://user-images.githubusercontent.com/200411/179867532-b21a8691-91fe-49c1-951b-ce58200b2d3e.png)

Currently the ROR field requires the user to strip off the numeric ID for the ROR to be accepted, which they will not know how to do (and the error message doesn't clarify):

![image](https://user-images.githubusercontent.com/200411/179867487-380dc4e1-dbc5-4fad-b767-aa5ed42a1974.png)

I would recommend we follow the lead of ORCiDs and store RORs as full URLs in the database. I've only skimmed the COUNTER code of practice, and it seems unclear about it, but I'll bet it expects full URLs instead of just the ID part.